test/qpy_compat/test_qpy.py

```diff
@@ -983,6 +1012,34 @@ def load_qpy(qpy_files, version_parts):
                 circuit, qpy_circuits[i], i, version_parts, bind=bind, equivalent=equivalent
             )
 
+    from qiskit.qpy.exceptions import QpyError
+
+    while pulse_files:
+        path, version = pulse_files.popitem()
+
+        if version_parts < version or version_parts >= (2, 0):
+            continue
```

Comment:
Just for any onlookers in the future, `version_parts` here is the version of qiskit used to generate the payloads being loaded.

Reply:
Will add a comment in #13662